clusterctl/examples/vsphere/machinedeployment.yaml.template

```diff
@@ -0,0 +1,34 @@
+apiVersion: "cluster.k8s.io/v1alpha1"
+kind: MachineDeployment
+metadata:
+  name: test-machineset-1
```

Comment:
This should also be updated to test-machinedeployment-1 :)

Reply:
doh
fixed.